packages/react-scripts/config/webpackDevServer.config.js

```diff
@@ -115,7 +115,7 @@ module.exports = function(proxy, allowedHost) {
       // We do this in development to avoid hitting the production cache if
       // it used the same host and port.
       // https://github.com/facebook/create-react-app/issues/2272#issuecomment-302832432
-      app.use(noopServiceWorkerMiddleware());
+      app.use(noopServiceWorkerMiddleware('/'));
```

Comment:
Looks like the build is failing. Maybe something else is also calling `noopServiceWorkerMiddleware`.

This is definitely a hack, but locally I've been changing that [service worker](https://github.com/shipt/create-react-app/blob/master/packages/react-dev-utils/noopServiceWorkerMiddleware.js#L12) to have this signature

```
module.exports = function createNoopServiceWorkerMiddleware(servedPath = '/') {
```

the default `'\'` is new.

Reply:
Ahh, I see. I updated that to see if that fixes the build.

Reply:
Still seems to be failing :(

Reply:
Yeah, I see that. [It looks like this build has been failing for some time now.](https://github.com/shipt/create-react-app/pull/4)

Reply:
Looks like the build is passing now. Man, these branches are a mess. I need to find out what the thinking was around having things separated on different branches.

In this case, it looks like the build script was looking for files (module.js/context.js) in the templates folder but those files exist on a different branch `@shipt/cra-template-sg1-module`.

Reply:
Oh man...